src/app/components/ATIAnalytics/amp/ampAnalyticsJson.test.ts

```diff
@@ -20,4 +20,90 @@ describe('AMP ATI Analytics', () => {
       triggers: { trackPageview: { on: 'visible', request: 'pageview' } },
     });
   });
+
+  describe('Reverb URL Helper - getAmpAnalyticsPageViewUrl', () => {
+    const mockReverbParams = {
+      params: {
+        page: {
+          contentId: 'urn:bbc:optimo:asset:cly1jw4x585o',
+          contentType: 'article',
+          name: 'gahuza.articles.cly1jw4x585o.page',
+          producer: 'GAHUZA',
+          additionalProperties: {
+            app_name: 'news-gahuza',
+            app_type: 'amp',
+            content_language: 'rw',
+            product_platform: null,
+            // eslint-disable-next-line no-template-curly-in-string
+            referrer_url: '${documentReferrer}',
+            // eslint-disable-next-line no-template-curly-in-string
+            x5: '${sourceUrl}',
+            x8: 'simorgh',
+            x9: "US%20irashishikariza%20u%20Rwanda%20na%20DR%20Congo%20kugera%20ku%20mahoro%20bikazana%20n'ishoramari%20rya%20miliyari%20z'amadorari",
+            x10: null,
+            x11: '2025-05-02T07:00:25.419Z',
+            x12: '2025-05-02T07:00:25.419Z',
+            x13: 'Democratic+Republic+of+Congo~Rwanda~March+23+Movement~M23+offensive',
+            x14: '3548f44f-46f5-4e6e-8628-3f668f161691~8125f2a9-3259-4f35-ab75-d9a6577fdc88~b03e7bfd-9a46-4053-aeed-f9f55f5e5567~c78c7532-43b3-490d-ad5b-0fc47b906e42',
+            x16: '',
+            x17: 'Democratic+Republic+of+Congo~Rwanda~March+23+Movement~M23+offensive',
+            x18: null,
+          },
+        },
+        user: {
+          isSignedIn: false,
+        },
+      },
+      eventDetails: {
+        eventName: 'pageView',
+      },
+    };
+
+    it.each`
+      environment | destination                 | baseUrl
+      ${'local'}  | ${'WS_NEWS_LANGUAGES_TEST'} | ${'https://logws1363.ati-host.net/hit.xiti?'}
+      ${'test'}   | ${'WS_NEWS_LANGUAGES_TEST'} | ${'https://logws1363.ati-host.net/hit.xiti?'}
+      ${'live'}   | ${'WS_NEWS_LANGUAGES'}      | ${'https://a1.api.bbc.co.uk/hit.xiti?'}
+    `(
+      'should match this JSON structure for environment - $environment',
+      ({ environment, destination, baseUrl }) => {
+        const reverbParams = {
+          ...mockReverbParams,
+          params: {
+            ...mockReverbParams.params,
+            env: environment,
+            page: {
+              ...mockReverbParams.params.page,
+              destination,
+            },
+          },
+        };
```

Comment:
Think if you cast this with `as unknown as ReverbBeaconConfig` it should resolve the type issues.